src/collective/opendata/plugins/content/__init__.py

```diff
@@ -27,10 +28,10 @@ class Content(DataPlugin):
     implements(IDataPlugin)
 
     name = 'content'
-    title = 'Content Metadata'
-    description = '''Content information'''
+    title = _(u'Content Metadata')
+    description = _(u'''Content information''')
 
-    portal_types = ['File', 'Document', 'Image', 'Event', 'News Item', 'Folder']
+    portal_types = [_(u'File'), _(u'Document'), _(u'Image'), _(u'Event'), _(u'News Item'), _(u'Folder')]
```

Comment:
those names are already translated in plone.app.locales, isn't it? thy don't just use that message factory?